docs: v1.20 - autoscaling cooldown support (#3782)
aws/copilot-cli: site/content/blogs/release-v120.en.md

    ✔ Successfully deployed pipeline: env-pipeline
    ```

## Autoscaling Cooldown Support
A small addition to our service manifests: the ability to configure autoscaling cooldown periods.
For `Load Balanced`, `Backend`, and `Worker` Services, you can now configure their autoscaling fields under `count` to have custom cooldown periods.

> Previously, each scaling metric such as `cpu_percentage` has a set in cooldown of 120 secs and out cooldown of 60 seconds. Now you can set a global cooldown period:

Reviewer: not sure if this was meant?
```suggestion
Previously, each scaling metric such as `cpu_percentage` had a fixed 'in' cooldown of 120 secs and 'out' cooldown of 60 seconds. Now, you can set a global cooldown period:
```

    ✔ Successfully deployed pipeline: env-pipeline
    ```

> ## Autoscaling Cooldown Support

Reviewer 2: add it to the top section as well! with a link down to here

A small addition to our service manifests: the ability to configure autoscaling cooldown periods.
For `Load Balanced`, `Backend`, and `Worker` Services, you can now configure this for their corresponding autoscaling fields.

To configure the cooldown field, your service needs some extra fields to allow the scaling to begin.

> First it needs a `range` of the number of ECS clusters your service can run, and then it needs at least one autoscaling field such as `cpu_percentage` to set a target which the service will scale to meet.

Reviewer 2: ```suggestion
First it needs a `range` of the number of ECS clusters your service can run, and then it needs at least one autoscaling field such as `cpu_percentage` to set a target the service will scale to meet.
```

> First it needs a `range` of the number of ECS clusters your service can run, and then it needs at least one autoscaling field such as `cpu_percentage` to set a target which the service will scale to meet.

Reviewer 3: I wonder if we can prune any of the text to make it a bit more terse, what do you think of something like?
> You can now configure the autoscaling fields under [`range`](../docs/manifest/lb-web-service.en.md#count-rage) to have custom cooldown periods.
> Previously, each scaling metric such as `cpu_percentage` has a set in cooldown of X secs and out cooldown of Y seconds. Now you can set a global cooldown period:
>  \# your yaml
> Alternatively, you can set individual cooldowns:

??? example "Using general autoscaling cooldowns"

    ```
    count:
      range: 1-10
      cooldown:
        in: 30s
        out: 30s
      cpu_percentage: 50
    ```

> This also gives you the option to specify the cooldowns for an individual field.

Reviewer 2: ```suggestion
You also have the option to specify the cooldowns for an individual field.
```
